libraries/freertos_plus/aws/ota/src/http/aws_iot_ota_http.c

```diff
@@ -1177,7 +1185,15 @@ OTA_Err_t _AwsIotOTA_CleanupData_HTTP( OTA_AgentContext_t * pAgentCtx )
     /* Unused parameters. */
     ( void ) pAgentCtx;
 
-    memset( &_httpDownloader, 0, sizeof( _httpDownloader_t ) );
+    /* Disconnect from the S3 server. */
+    IotHttpsClient_Disconnect( _httpDownloader.httpConnection.connectionHandle );
```

Comment:
For possible warnings?
```suggestion
    ( void ) IotHttpsClient_Disconnect( _httpDownloader.httpConnection.connectionHandle );
```